src/main/resources/init_db.sql

```diff
@@ -18,3 +18,19 @@ CREATE TABLE `drivers` (
                                   PRIMARY KEY (`id`),
                                   UNIQUE INDEX `id_UNIQUE` (id ASC) VISIBLE,
                                   UNIQUE INDEX `license_number_UNIQUE` (`license_number` ASC) VISIBLE);
+
+DROP TABLE IF EXISTS `cars`;
+CREATE TABLE `cars` (
+                        `id` BIGINT NOT NULL AUTO_INCREMENT,
+                        `model` VARCHAR (255) DEFAULT NULL,
+                        `is_deleted` TINYINT NOT NULL DEFAULT 0,
+                        `manufacturer_id` BIGINT DEFAULT NULL,
+                        PRIMARY KEY (`id`),
+                        CONSTRAINT `cars_manufacturers_fk` FOREIGN KEY (`manufacturer_id`) REFERENCES `manufacturers` (`id`));
+
+DROP TABLE IF EXISTS `cars_drivers`;
```

Comment:
it is better to first drop all tables in the correct order, and then create them